src/components/structures/RightPanel.tsx

```
@@ -159,6 +163,10 @@ export default class RightPanel extends React.Component<Props, IState> {
     private onSearchQueryChanged = (searchQuery: string): void => {
         this.setState({ searchQuery });
     };
+    private handleResize = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)